internal/convert.go

```diff
@@ -373,38 +419,66 @@ func (conv *Conv) getStatementStat(s string) *statementStat {
 	return conv.stats.statement[s]
 }
 
-func (conv *Conv) skipStatement(l []nodes.Node) {
-	if conv.schemaMode() { // Record statement stats on first pass only.
+func (conv *Conv) SkipStatement(l []nodes.Node) {
+	if conv.SchemaMode() { // Record statement stats on first pass only.
 		s := prNodes(l)
 		VerbosePrintf("Skipping statement: %s\n", s)
 		conv.getStatementStat(s).skip++
 	}
 }
+func (conv *Conv) SkipStatementMySQL(stmtType string) {
+	if conv.SchemaMode() {
+		VerbosePrintf("Skipping statement: %s\n", stmtType)
+		conv.getStatementStat(stmtType).skip++
+	}
+}
 
-func (conv *Conv) errorInStatement(l []nodes.Node) {
-	if conv.schemaMode() { // Record statement stats on first pass only.
+func (conv *Conv) ErrorInStatement(l []nodes.Node) {
+	if conv.SchemaMode() { // Record statement stats on first pass only.
 		s := prNodes(l)
 		VerbosePrintf("Error processing statement: %s\n", s)
-		conv.getStatementStat(s).error++
+		conv.getStatementStat(s).Error++
+	}
+}
+func (conv *Conv) ErrorInStatementMYSQL(stmtType string) {
+	if conv.SchemaMode() { // Record statement stats on first pass only.
+		VerbosePrintf("Error processing statement: %s\n", stmtType)
+		conv.getStatementStat(stmtType).Error++
 	}
 }
 
-func (conv *Conv) schemaStatement(l []nodes.Node) {
-	if conv.schemaMode() { // Record statement stats on first pass only.
+func (conv *Conv) SchemaStatement(l []nodes.Node) {
+	if conv.SchemaMode() { // Record statement stats on first pass only.
 		conv.getStatementStat(prNodes(l)).schema++
 	}
 }
+func (conv *Conv) SchemaStatementMySQL(stmtType string) {
+	if conv.SchemaMode() { // Record statement stats on first pass only.
+		conv.getStatementStat(stmtType).schema++
+	}
+}
 
-func (conv *Conv) dataStatement(l []nodes.Node) {
-	if conv.schemaMode() { // Record statement stats on first pass only.
+func (conv *Conv) DataStatement(l []nodes.Node) {
+	if conv.SchemaMode() { // Record statement stats on first pass only.
 		conv.getStatementStat(prNodes(l)).data++
 	}
 }
 
-func (conv *Conv) schemaMode() bool {
+func (conv *Conv) SchemaMode() bool {
 	return conv.mode == schemaOnly
 }
 
-func (conv *Conv) dataMode() bool {
+func (conv *Conv) DataMode() bool {
 	return conv.mode == dataOnly
 }
+
+func byteSize(r *row) int64 {
```

Comment:
Just noting that byteSize used to be defined in data.go and used in convert.go.
In short, this is a cleanup, and allows us to keep byteSize private.